packages/frontend/src/components/composer/ComposerMessageInput.tsx

```diff
@@ -124,17 +123,14 @@ export default class ComposerMessageInput extends React.Component<
       this.resizeTextareaAndComposer()
       if (prevState.text !== this.state.text) {
         if (!this.state.loadingDraft) {
-          this.saveDraft()
+          this.saveDraft(this.state.text, prevState.chatId)
```

Comment:
Why use `prevState.chatId` here instead of `this.state.chatId`? They're the same inside this `if` block.